app/views/admin/_adminheader.html.erb

```diff
@@ -11,6 +11,9 @@
         <li class="nav-item">
           <%= link_to "Kudos index", admin_kudos_path, class: "nav-link" %>
         </li>
+        <li class="nav-item">
+          <%= link_to "Company value", admin_company_values_path, class: "nav-link" %>
```

Comment:
I think plural form will be better here -  _Company Values_, also consider renaming _Kudos index_ to _Kudos_, but this si cosmetics